openvdb/openvdb/Platform.h

```diff
@@ -54,6 +52,25 @@
     #endif
 #endif
 
+/// Windows defines
+#ifdef _WIN32
+    // Math constants are not included in <cmath> unless _USE_MATH_DEFINES is
+    // defined on MSVC
+    // https://docs.microsoft.com/en-us/cpp/c-runtime-library/math-constants
+    #ifndef _USE_MATH_DEFINES
+        #define _USE_MATH_DEFINES
+    #endif
+    ///Disable the non-portable Windows definitions of min() and max() macros
+    #ifndef NOMINMAX
+        #define NOMINMAX
+    #endif
```

Comment:
We discussed moving these defines into the headers as opposed to the CMake build system as they are **required** by VDB for successful builds. This further removes the need for downstream projects to define them